domain/tutoringModel/NestorFolder/nestor_utils.py

```diff
@@ -0,0 +1,158 @@
+# import pandas as pd
+# import matplotlib.pyplot as plt
+# import networkx as nx
+# from pgmpy.estimators import BDeuScore
+from pgmpy.estimators import ExpectationMaximization as EM
+
+# from pgmpy.estimators import HillClimbSearch, MmhcEstimator, TreeSearch
+from pgmpy.models import BayesianNetwork
+
+# from pgmpy.estimators import PC, ExhaustiveSearch, K2Score
+
+
+def categorize_lisk_bfi(input_dataframe, encoding_columns):
+    """
+    This function decodes the nuerical responses of list-k and bfi
+    to categories and suit PGMPY inference methods
+    """
+    likert_map = {
+        1: "strong_disagree",
+        2: "disagree",
+        3: "neither_agree_disagree",
+        4: "agree",
+        5: "strong_agree",
+    }
+    # looping through rows
+    for each_row in range(len(input_dataframe)):
```

Comment:
can shorten it a lot and increase readability.
```suggestion
    for col in encoding_columns:
        input_dataframe[col] = input_dataframe[col].map(likert_map)
```